libs/solid/slideshow/src/lib/components/slideshow-select/slideshow-select.component.html

```diff
@@ -5,9 +5,17 @@
     [routerLink]="[slideshow.id]"
   >
     <img
+      *ngIf="slideshow.title_image !== null; else NoImg"
       [src]="slideshow.title_image.img.thumbnail"
       [alt]="slideshow.title_image.img_alt"
     />
+    <ng-template #NoImg>
+      <img
+        src="assets/icons/icon.svg"
+        alt="No image available"
```

Comment:
As long as we do not have a localized app, we should write this in German: "Kein Bild vorhanden"
(I know it was already wrong in the issue, so: my fault. But please fix it anyway 😉 )